File: swimming-contest-software/Model/src/main/java/model/protocol/FindAllSwimmersDetailsForRaceResponse.java
package model.protocol;

import model.domain.dtos.SwimmerDTO;

import java.util.List;

public class FindAllSwimmersDetailsForRaceResponse implements Response {
    private final List<SwimmerDTO> allSwimmersDetailsForRace;

    public FindAllSwimmersDetailsForRaceResponse(List<SwimmerDTO> allSwimmersDetailsForRace) {
        this.allSwimmersDetailsForRace = allSwimmersDetailsForRace;
    }

    public List<SwimmerDTO> getAllSwimmersDetailsForRace() {
        return allSwimmersDetailsForRace;
    }
}

File: Model/Protocol/Responses/FindAllSwimmersDetailsForRaceResponse.cs
using System;
using System.Collections.Generic;
using Model.Domain.DTOs;

namespace Model.Protocol.Responses;

[Serializable]
public class FindAllSwimmersDetailsForRaceResponse : IResponse
{
    public FindAllSwimmersDetailsForRaceResponse(List<SwimmerDTO> allSwimmersDetailsForRace)
    {
        AllSwimmersDetailsForRace = allSwimmersDetailsForRace;
    }

    public List<SwimmerDTO> AllSwimmersDetailsForRace { get; }
}